### PR TITLE
Refactor openAQ

### PR DESCRIPTION
**Pull request by purple-waves**
Refactor to calculate the aqi more accurately

### CODE REVIEW

File: src/main/java/com/google/sps/data/OpenAQMeasurement.java [hunk 1]
```diff
@@ -8,72 +10,26 @@
...
+  private HashMap<String, Double> createStandardsMap() {
+    HashMap<String, Double> map = new HashMap<>();
+    map.put("o3_ppm", 0.1);
+    map.put("no2_ppm", 0.12);
+    map.put("so2_ppm", 0.2);
+    map.put("co_ppm", 9.0);
+    map.put("pm10_µg/m³", 50.0);
+    map.put("pm25_µg/m³", 25.0);
+    return map;
   }
```
Comment (roroross) [inline src/main/java/com/google/sps/data/OpenAQMeasurement.java:25]: what are the advantages of refactoring it into a map over what was there before?

Comment (purple-waves) [reply src/main/java/com/google/sps/data/OpenAQMeasurement.java:25]: It's easier to interpret than a giant switch statement.

Also, this was a whole new way of calculating the AQI that required rewriting code, and last time I didn't think of using a map with `map.getOrDefault` to only use measurements that I could turn into an AQI

Comment (purple-waves) [reply src/main/java/com/google/sps/data/OpenAQMeasurement.java:25]: To clarify the purpose of this PR a bit, in the old version I would calculate the AQ category, then the AQI would be the midpoint of the category's threshold, which wasn't very useful or interesting data wise.

Now, it calculates the AQI for each individual measurement (which I didn't realise was a thing before) (i.e. pm10, pm25, co, no2) then works out the maximum for each station, no need for categories.

Comment (roroross) [reply src/main/java/com/google/sps/data/OpenAQMeasurement.java:25]: ah, that makes more sense, thanks!

**Review by roroross — APPROVED** [review]

**Review by sbland-g — APPROVED** [review]
This is a much nicer solution for the AQI calculation.

**Pull request merged**